MainWindow.xaml

```diff
@@ -1,17 +1,17 @@
-﻿<Window x:Class="Pikuniku_Save_Manager.MainWindow"
+<Window x:Class="Pikuniku_Save_Manager.MainWindow"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:Pikuniku_Save_Manager"
         mc:Ignorable="d"
-        Title="Pikuniku Save Manager" Height="220" Width="600" ResizeMode="NoResize">
+        Title="Pikuniku Save Manager" Height="220" Width="600" ResizeMode="CanResize">
     <Grid>
-        <ComboBox x:Name="save" Margin="105,30,274,123"/>
-        <Label Content="Save Selection:&#xD;&#xA;" HorizontalAlignment="Left" Margin="10,30,0,0" VerticalAlignment="Top"/>
-        <Button Content="..." HorizontalAlignment="Left" Margin="309,30,0,0" VerticalAlignment="Top" Height="28" Width="31" Click="Button_Click"/>
-        <Button x:Name="download" Content="Download Saves" HorizontalAlignment="Left" Margin="369,30,0,0" VerticalAlignment="Top" Height="28" Width="170" Click="Download_Click"/>
-        <ComboBox x:Name="res" Margin="105,85,274,68">
+        <ComboBox x:Name="save" Margin="120,30,255,123"/>
+        <Label Content="Save Selection:&#xD;&#xA;" HorizontalAlignment="Left" Margin="30,32,0,0" VerticalAlignment="Top"/>
+        <Button Content="..." HorizontalAlignment="Left" Margin="330,30,0,0" VerticalAlignment="Top" Height="30" Width="30" Click="Button_Click"/>
```

Comment:
This button was still off by a couple pixels both vertically size-wise and horizontally position-wise on both my PC and another PC I tested on, unlike in the screenshot you sent.

Reply:
Interesting, I'll see if I can fix it.